### ISSUE TITLE
Quick code fix for latest SDK and Tablet version please

### ISSUE THREAD

**Issue by Termitebug** [jklingen/space-inspector#17]
Hi, please edit fileworker.h and fileworker.ccp to following so that compiler will complete without errors:

(fileworker.h line 39-40)
```
...
protected:
    void run() Q_DECL_OVERRIDE;
...
```

(fileworker.ccp line 77-78)
```
...
void FileWorker::run()
{
...
```

Works fine on Jolla Tablet 2.1.0.11. I compiled it with latest beta SDK.
So can you upload your app to Harbour (Jolla Store) or Warehouse also as a i486.rpm version, please :) ?

**Comment by jklingen** [comment]
Hey there, thanks for providing this helpful piece of information and sorry for the late response :astonished: 
Finally found the time to get my SFOS dev environment up and running again and take care of this. Will release a new version soon (here and in Jolla Store), which will also include an i486 build.

Thanks again for your support!

**Comment by Olf0** [comment]
Thanks @jklingen!
And IMO this issue can be closed now that SpaceInspector 0.6 has been released in the Jolla Store.

**Comment by jklingen** [comment]
Alright, thanks a lot 😄
